packages/fast-components-react-msft/src/number-field/README.md

```diff
@@ -1,11 +1,7 @@
-# Label
+# Number field
 
 A *number field* is a form component which allows users to input and edit numerical data.
 
 ## Usage
 
-Use sentence case (e.g. "Last name"), but avoid punctuation such as colons. *Label* text should be short and succinct, not a full sentence. When labeling components, write the *label* text as a noun or a concise noun phrase, without using full sentences. Using commas to separate list items is an exception to this rule.
-
-## Behavioral guidance
-
-Use *number field* when numeric values of input are required. Use step to control number accuracy. It is recommended to have a label above the number field to indicate its usage.
+The *number field* is not appropriate for input that happens to only consist of numbers but isn’t strictly speaking a number. For example, it would be inappropriate for credit card numbers or US postal codes. A simple way of determining whether to use the *number field* is to consider whether it would make sense for the input control to have a spinbox interface (e.g., with “up” and “down” arrows). When a number field is not appropriate, use text field instead.
```

Comment:
I feel like this might be more clear if we focus first on when to use it as opposed to when not to use it.  Then maybe provide a couple actual examples if at all possible.  What is a spinbox interface?  I'm not actually clear when to use it by reading the documentation.

Reply:
I think this is good feedback. I always have appreciated when we've stated things from a "Use `component` when..." and then followed with, "don't use this for _____". It just flows better and gives me context for the acceptable use case right away

Reply:
@chrisdholt & @awentzel how does this read now?

Reply:
Much more clear and has a more positive feel to it (I get it, that sounds weird)...but I think the recent push is much better and gives me positive context on when and how to use appropriately.